sblib/inc/sblib/platform.h

```diff
@@ -56,14 +56,15 @@ unsigned int* ioconPointer(int port, int pinNum);
   extern unsigned char FLASH[];
 # define __vectors_start__ *FLASH
 #else
-  extern const unsigned int __vectors_start__;
+#if not defined LPC_FLASH_BASE
```

Comment:
Think it should be
like 
# define LPC _Flash_Base *FLASH
In iap emulation case